core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -2877,7 +2891,10 @@ disputeSummaryWindow.close.msg=Ticket closed on {0}\n\
 disputeSummaryWindow.close.msgWithSig={0}{1}{2}{3}
 
 disputeSummaryWindow.close.nextStepsForMediation=\nNext steps:\n\
-Open trade and accept or reject suggestion from mediator
+  Open trade to accept suggestion from mediator. If you don't agree with it, reopen the ticket and explain your motives. \
```

Comment:
Open trade to accept suggestion from mediator, or click Reject Proposal If you don't agree with it. Explain why do you disagree at the mediation ticket.

Reply:
Change made, screenshot updated above in OP.

Reply:
New suggestion for "Next steps" field:

> Go to Open trades to accept the suggestion from mediator, or click Reject Proposal if you disagree. Explain your reasons at the mediation ticket. 
If the trade has not concluded in the next 48h after accepting mediation proposal, inform your mediator on this chat.
Keep in mind that if arbitration is opened by any party, it will cost 1 security deposit. The full amount can't be reimbursed for security reasons.